parley/src/layout/editor.rs

```diff
@@ -49,122 +73,125 @@ where
             selection: Default::default(),
             cursor_mode: Default::default(),
             width: Default::default(),
-            scale: Default::default(),
+            scale: 1.0,
+            layout_dirty: Default::default(),
+            // We don't use the `default` value to start with, as our consumers
+            // will choose to use that as their initial value, but will probably need
+            // to redraw if they haven't already.
+            generation: Generation(1),
```

Comment:
With this added `Generation::default` is fine to keep. The choice is purely a semantic one now.